source/Calamari/Calamari.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <Reference Include="WindowsBase" />
     <PackageReference Include="Calamari.AzureScripting" Version="8.2.0" />
+    <PackageReference Include="Calamari.Scripting" Version="8.2.0" />
```

Comment:
Just updating `Calamari.AzureScripting` isn't enough, we need to reference `Calamari.Scripting` explicitly